Gemfile.lock

```diff
@@ -46,18 +75,36 @@ GEM
       terminal-table
     simplecov-html (0.12.3)
     simplecov_json_formatter (0.1.3)
+    sinatra (2.1.0)
+      mustermann (~> 1.0)
+      rack (~> 2.2)
+      rack-protection (= 2.1.0)
+      tilt (~> 2.0)
+    sinatra-contrib (2.1.0)
+      multi_json
+      mustermann (~> 1.0)
+      rack-protection (= 2.1.0)
+      sinatra (= 2.1.0)
+      tilt (~> 2.0)
     terminal-table (3.0.1)
       unicode-display_width (>= 1.1.1, < 3)
+    tilt (2.0.10)
     unicode-display_width (2.0.0)
+    xpath (3.2.0)
+      nokogiri (~> 1.8)
 
 PLATFORMS
   ruby
 
 DEPENDENCIES
+  capybara
+  pg
   rspec
   rubocop (= 1.20)
   simplecov
   simplecov-console
+  sinatra
+  sinatra-contrib
 
 RUBY VERSION
    ruby 3.0.2p107
```

Comment:
Clear division of gems, dependencies and platforms -
Maybe divide gems into test groups? See http://bundler.io/groups.html for more details. [from code review rubric]